rust-runtime/aws-smithy-http-server/examples/pokemon-service/tests/simple_integration_test.rs

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+// Files here are for running integration tests.
+// These tests only have access to your crate's public API.
+// See: https://doc.rust-lang.org/book/ch11-03-test-organization.html#integration-tests
+
+use std::time::Duration;
+
+use crate::helpers::{client, PokemonService};
+use tokio::time;
+
+#[macro_use]
+mod helpers;
+
+#[tokio::test]
+async fn simple_integration_test() {
+    let _program = PokemonService::run();
+    // Give PokemonService the time to start up.
+    time::sleep(Duration::from_millis(50)).await;
+
+    let service_statistics_out = client().get_server_statistics().send().await.unwrap();
+    assert_eq!(0, service_statistics_out.calls_count.unwrap());
+
+    let pokemon_species_output = client().get_pokemon_species().name("pikachu").send().await.unwrap();
+    assert_eq!("pikachu", pokemon_species_output.name().unwrap());
+
+    let service_statistics_out = client().get_server_statistics().send().await.unwrap();
+    assert_eq!(1, service_statistics_out.calls_count.unwrap());
+
+    let pokemon_species_error = client()
+        .get_pokemon_species()
+        .name("some_pokÃ©mon")
```

Comment:
```suggestion
        .name("some_pokémon")
```

Reply:
I am speechless.. How the heck did I make this Ã© thing??